fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -885,23 +885,27 @@ private FDBDatabaseRunner getRunner() {
     @API(API.Status.UNSTABLE)
     public static class Config {
         private final int maxLimit;
+        private final int initialLimit;
         private final int maxWriteLimitBytes;
         private final int maxRetries;
         private final int recordsPerSecond;
         private final long progressLogIntervalMillis;
         private final int increaseLimitAfter;
         private final long timeLimitMilliseconds;
+        private final long transactionTimeQuotaMilliseconds;
```

Comment:
It's a bit awkward because the `timeLimitMilliseconds` field already exists, but I think `transactionTimeLimitMilliseconds` is more consistent. I think it's probably a straightforward enough distinction for users that `timeLimitMilliseconds` controls how long the entire build has and `transactionTimeLimitMilliseconds` controls how often it commits an individual transaction, which I think is straightforward enough

Reply:
✅ 

I thought that time limit is semantically different than time quota - as the first implies stopping and the second continuation (like schedulers' time quota). But I also can see the value in uniformity.